### DISCUSSION TITLE
Create a photo gallery for posters and graphics

### DISCUSSION THREAD

**Discussion by siavashgm** [WeAreMahsaAmini/Core#220] [Ideas]
Many people around the world need high quality graphic materials that they can print as posters and bring to protests or they might even print on t-shirt. 
It might be a good idea to create a website where these materials will be available and easily accessible globally so that people can download them as use them for their needs. 
These websites needs to be SEO friendly so that users can find these materials easily in google search.
The graphic materials should also be easily shareable in social media (IG, Twitter) so that we can benefit from network effort.

**Comment by SamanZargarzadeh** [comment]
I think we can combine this idea of yours with #428. I don't have a major in website design, but I can help to collect designs and create an archive. I also know Photoshop, which might be useful for mixing designs for printing t-shirts and placards.
